test/jdk/java/text/Format/DateFormat/DateFormatTest.java

```diff
@@ -342,12 +343,17 @@ void tryPat994(SimpleDateFormat format, String pat, String str, Date expected)
     // Test pattern with runs things together
     public void TestRunTogetherPattern985()
     {
+        Date date1 = new Date();
+        var defZone = ZoneId.systemDefault();
+        if (defZone.getRules().getTransition(date1.toInstant().atZone(defZone).toLocalDateTime()) != null) {
+            logln("At the offset transition. Round trip test skipped.");
```

Comment:
Should the message be more inlined with the test name vs indicate Round trip test is skipped?

Reply:
I changed the test not to skip even at the overlap, by explicitly giving the zone pattern. Unfortunately, this cannot be applied to the other location in `NonGregorianFormatTest`, because the argument is a bare `DateFormat`.